deep_field_metadetect/jaxify/tests/test_jax_deep_metacal.py

```diff
@@ -0,0 +1,339 @@
+import multiprocessing
+
+import jax.numpy as jnp
+import numpy as np
+import pytest
+
+from deep_field_metadetect.jaxify.jax_metacal import (
+    jax_metacal_op_shears,
+    jax_metacal_wide_and_deep_psf_matched,
+)
+from deep_field_metadetect.jaxify.observation import NT_to_ngmix_obs
+from deep_field_metadetect.utils import (
+    MAX_ABS_C,
+    MAX_ABS_M,
+    assert_m_c_ok,
+    estimate_m_and_c,
+    fit_gauss_mom_mcal_res,
+    make_simple_sim,
+    measure_mcal_shear_quants,
+    print_m_c,
+)
+
+
+def _run_single_sim(
+    seed,
+    s2n,
+    g1,
+    g2,
+    deep_noise_fac,
+    deep_psf_fac,
+    skip_wide,
+    skip_deep,
+):
+    obs_w, obs_d, obs_dn = make_simple_sim(
+        seed=seed,
+        g1=g1,
+        g2=g2,
+        s2n=s2n,
+        deep_noise_fac=deep_noise_fac,
+        deep_psf_fac=deep_psf_fac,
+        return_NT=True,
+    )
+    mcal_res = jax_metacal_wide_and_deep_psf_matched(
+        obs_w,
+        obs_d,
+        obs_dn,
+        dk_w=2 * jnp.pi / (53 * 0.2) / 4,
+        dk_d=2 * jnp.pi / (53 * 0.2) / 4,
```

Comment:
Can you comment on what the various factors here are? Is the factor of 53 related to the PSF size or the image size or do they happen to be the same?